glean-core/preview/src/lib.rs

```diff
@@ -207,5 +207,23 @@ pub fn submit_ping_by_name(ping: &str, reason: Option<&str>) -> bool {
     with_glean(|glean| glean.submit_ping_by_name(ping, reason).unwrap_or(false))
 }
 
+/// Add a debug tag to pings.
+///
+/// See `glean_core::Glean.set_debug_view_tag`.
+///
+/// ## Return value
+///
+/// Returns true if the tag was set, false otherwise.
+pub fn set_debug_view_tag(value: &str) -> bool {
```

Comment:
Do we need a public facing API for these? On Android and Swift, we expose other APIs to tag pings. On non-mobile platforms, we'll add environment variable detection within the SDK itself. I'd rather not provide an ad-interim public API

Reply:
That's I good question. I am not sure really, I added this to the preview like  added it to other bindings and just didn't think about it very deeply. If we are going to set debug options through enviroment variables only, it makes more sense to just remove the commit for the preview changes from this PR altogether and wait for when I work on getting this options from the environment.

Reply:
We'll allow both: on Android and iOS, we'll support both environment variable and command-line based workflows